app/assets/stylesheets/ngao-footer.scss

```diff
@@ -0,0 +1,37 @@
+@import "iu_variables";
+
+#ngao-footer {
+  width: 100%;
+  margin: 0 auto;
+  background-color: $iu-mahogany;
+  .row {
```

Comment:
rather than override the boostrap class (.row), could you add a custom class and apply the styling to that.